app/pages/Accounts/AccountDetailsPage/CredentialInformation/DisplayIdentityAttributes.tsx

```diff
@@ -10,22 +11,21 @@ import {
 const attributeNames: Record<string, string> = attributeNamesJson;
 
 interface Props {
-    credential: CredentialDeploymentValues;
+    revealedAttributes: Record<AttributeKey, string>;
 }
 
 /**
  *  DIsplays the revealed Attributes of the chosen account.
  *  TODO: Use local credential to get attributes?
  */
 export default function DisplayIdentityAttributes({
-    credential,
+    revealedAttributes,
 }: Props): JSX.Element | null {
-    const attributes = credential.policy.revealedAttributes;
-    const attributeKeys = Object.keys(attributes);
+    const attributeKeys = Object.keys(revealedAttributes);
 
     if (attributeKeys.length === 0) {
         return (
-            <div className="pT10" key={credential.credId || credential.regId}>
+            <div className="pT10">
```

Comment:
I'm guessing you will now get an error if you display 2 credentials on an account, where neither has a revealed credential. So you probably need to add some other key.

Reply:
This shouldn't have a key, because it is not the responsibility of this component to provide a key, but the "list"-component. 
And this is used is not used directly in a list, so we shouldn't get an error.